Mathlib/Order/Filter/Basic.lean

```diff
@@ -1734,6 +1733,8 @@ instance : Trans ((· ≤ᶠ[l] ·) : (α → β) → (α → β) → Prop) (·
 
 end Preorder
 
+variable {l : Filter α}
```

Comment:
Maybe you add here `s` and `t`?

Reply:
I've checked: the file uses s and t for various things; there's no consistent pattern.
I think it's more readable as-is. (Refactoring that file is a good follow-up idea, though I'd probably start by splitting the file...)